http-server-jetty/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ dependencies {
     implementation(libs.jetty.servlet)
     implementation(libs.jetty.http2.server)
     implementation(libs.jetty.alpn.server)
+    implementation(libs.logback.access.jetty12)
```

Comment:
this should be `compileOnly` scope IMO since if someone doesn't use logback this introduces coupling